Project_2/djangoProject/cars/migrations/0003_auto_20220801_1046.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.0.6 on 2022-08-01 07:46
+import json
+
+from django.db import migrations, IntegrityError
+
+
+def refresh_data(apps, schema_editor):
+    Car = apps.get_model('cars', 'Car')
+    with open(r'C:\Users\Bogdan\PycharmProjects\VictorySquarePartners\Project_2\djangoProject\cars\templates\cars\hutto.json') as json_file:
+        vin_list = json.load(json_file)
+    for _ in vin_list:
+        try:
+            Car.objects.create(vin=_)
+        except IntegrityError:
+            pass
```

Comment:
If the migration does not work because of this, we should at least see a message that the car could not be created for the respective VIN